### DISCUSSION TITLE
Docker engine logs to syslog?

### DISCUSSION THREAD

**Discussion by airtower-luna** [moby/moby#48260] [Q&A]
Is there any way to get the engine logs to syslog *without* capturing stderr using some external process and forwarding from there (which is what the init script does)?

I'm emphasizing *engine* because there's plenty of documentation about log drivers for container logs, but the engine output seems to be stderr or nothing, which seems very odd.

**Comment by thaJeztah** [comment]
No, there currently isn't; the engine logs to stdout/stderr


> which is what the init script does

You mean the systemd unit, or one of the other init scripts?

**Reply by airtower-luna** [reply]
I mean the SysV init script: https://github.com/moby/moby/blob/50c3d19179e69f9e7ff01f688c4dbf32c5129ced/contrib/init/sysvinit-debian/docker

If the system I'm using was using systemd this wouldn't be an issue, then I could have journald capture the unit stdout/stderr. But on an embedded system with Busybox-init that's not an option, and Busybox' `start-stop-daemon` doesn't support `--no-close` to at least redirect to a file either.
